aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -335,7 +338,8 @@ protected ProgressEvent<ResourceModel, CallbackContext> removeAssociatedRoles(
             final AmazonWebServicesClientProxy proxy,
             final ProxyClient<RdsClient> proxyClient,
             final ProgressEvent<ResourceModel, CallbackContext> progress,
-            final Collection<DBClusterRole> roles
+            final Collection<DBClusterRole> roles,
+            final boolean recoveryMode
```

Comment:
nit: might be worth to keep this consistent with other use cases and name this "isRollback"

Reply:
Fixed. Thx.